.github/workflows/build.yml

```diff
@@ -8,44 +8,87 @@ on:
   workflow_dispatch:
     inputs:
       logLevel:
-        description: 'Reason'     
+        description: 'Reason'
         required: false
         default: 'Update package'
 
 jobs:
   build:
     name: build
     runs-on: ubuntu-latest
-    permissions:
-      id-token: write
-      attestations: write
     strategy:
       matrix:
         target_arch: [x86_64, x86, arm64-v8a, armeabi-v7a]
       fail-fast: false
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
+      - name: Checkout repo
+        uses: actions/checkout@v4
         with:
           submodules: 'true'
 
-      - uses: nttld/setup-ndk@v1
+      - name: Setting up NDK
+        uses: nttld/setup-ndk@v1
         id: setup-ndk
         with:
-          ndk-version: r23c
+          ndk-version: r27c
           add-to-path: false
-      - run: ./build.sh ${{ matrix.target_arch }}
-        env:
-          NDK_TOOLCHAIN: ${{ steps.setup-ndk.outputs.ndk-path }}/toolchains/llvm/prebuilt/linux-x86_64
 
-      - name: Attest build provenance
```

Comment:
Why was this step removed?

Reply:
It is moved to the release section in order to attest binaries only when a a release is push. https://github.com/Taknok/aapt2/blob/main/.github/workflows/build.yml#L84

It is due to how the google-please-release operates. This point will be null and removed as in others comments you express that you want that the release system be base and unified on what is already present in revanced.